code/src/App.js

```diff
@@ -1,9 +1,52 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, createStore } from '@reduxjs/toolkit';
+// removed configureStore
+
+import todos from './reducers/todos';
+
+import AddTodo from 'components/AddTodo';
+import TodoList from 'components/TodoList';
+import Summary from 'components/Summary';
+import Header from 'components/Header';
+
+const reducer = combineReducers({
+  todos: todos.reducer,
+});
+
+// Retrieve localstorage as initial state
+const persistedStateJSON = localStorage.getItem('todosReduxState');
```

Comment:
I understand what this localStorage does thanks to the comment, but it feels like the variable could have a more intuitive name?